guides/Getting-Started.md

```diff
@@ -77,6 +77,7 @@ Open PSCi by typing `spago repl` at the command line. Optionally, you can create
 
 As the introduction indicates, you can type `:?` to see a list of commands:
 
+    ```
```

Comment:
The indentation lets this be a code block, so the backtics aren't necessary. The rest of the file only uses them for language blocks.